capa/main.py

```diff
@@ -862,6 +870,22 @@ def install_common_args(parser, wanted=None):
                 choices=(BACKEND_VIV,),
                 default=BACKEND_VIV,
             )
+    
+    if "os" in wanted:
+        oses = [
+            (OS_AUTO, "detect OS automatically - default"),
+            (OS_LINUX,),
+            (OS_MACOS,),
+            (OS_WINDOWS,),
+        ]
+        os_help = ", ".join(["%s (%s)" % (o[0], o[1]) if len(o) == 2 else o[0] for o in oses])
+        parser.add_argument(
+            "-o",
+            "--os",
```

Comment:
```suggestion
            "--os",
```

recommend not using `-o` because that commonly means "output" in other tools